src/middlewared/middlewared/plugins/network.py

```diff
@@ -1075,6 +1075,12 @@ async def _common_validation(self, verrors, schema_name, data, itype, update=Non
                     'Failover needs to be disabled to perform network configuration changes.'
                 )
 
+                if len(data.get('failover_vhid', -1) <= 0 or > 255:
```

Comment:
How about the validators in the schema, using Min and Max?
Or is this better for some reason? I dont really know

Reply:
I've simply forgotten about those validators actually. Much better approach, I'll fix.